### DISCUSSION TITLE
iOS Notification error

### DISCUSSION THREAD

**Discussion by idkwhutimdoing** [blakeblackshear/frigate#10367] [General]
Using the notification blueprint and a Cloudflare tunnel to access HA. Cannot get a proper notification for the life of me. Always getting some error where the thumbnail or snapshot should be.

![image](https://github.com/blakeblackshear/frigate/assets/151402302/1a603cd2-db96-4e30-aafe-4112cfdddffd)

**Comment by templeowls21** [comment]
were you ever able to figure this out? I'm running into the same issue

**Reply by idkwhutimdoing** [reply]
Ended up having to adjust MQTT settings
